docs/userguide/configuration.rst

```diff
@@ -151,6 +151,7 @@ have been moved into a new  ``task_`` prefix.
 ``CELERY_TASK_TRACK_STARTED``              :setting:`task_track_started`
 ``CELERY_TASK_REJECT_ON_WORKER_LOST``      :setting:`task_reject_on_worker_lost`
 ``CELERYD_TIME_LIMIT``                     :setting:`task_time_limit`
+``CELERY_ALLOW_ERROR_CB_ON_CHORD_HEADER``  :setting:`task_allow_error_cb_on_chord_header`
```

Comment:
```suggestion
  :setting:`task_allow_error_cb_on_chord_header`
```

Reply:
I don't think we need the uppercase setting equivalent here, do we?

Reply:
For now it's better to have it to keep consistency.